scripts/build/Platform/Windows/build_config.json

```diff
@@ -355,13 +355,16 @@
     "TAGS": [
```

Comment:
Wouldn't that only show up as `installer` in the Jenkins pipeline UI? Not sure if we want to extend that later if we do other OS installers

No problem changing that otherwise

Reply:
@amzn-changml that was the thinking behind why "windows_" was included in the job name.  It also matches the packaging jobs naming convention.

Would something like "installer_vs2019" to match the other windows jobs be an acceptable compromise if a change happens?

Reply:
`installer_vs2019` sounds good
the pipeline UI has the platform on the "row". For other things like MARS we always send the platform name. The logs also have the platform visible.

The packaging jobs dont need that either... they were just ported from the old pipeline and they should go away once we have installers. The packaging jobs were meant to be temporary.

The reason to not have the "windows_" there is that when you override the list of jobs to build, you pick the platform and in the list of jobs you want to indicate something that is cross-platform. So, for example, if we have installer for all platforms, you want to select "Windows, Linux, Mac" and then put a job name like "installer" to trigger it in all of them. With the "windows_" you would have to pick the 3 platforms and then pass `windows_installer,mac_installer,linux_installer`

I will remove the `_vs2019`, we will have the `vsXXXX` only for the non-default one, when needed. We needed it becasue we had vs2017. Once vs2022 becomes official, we can add them with `_vs2022`, and leave the default one without the `_vs2019`. If we make 2022 the default one, we would make them not have the `_vs2022` and make all the 2019 have `_vs2019`.
The goal here is that we can define the list of jobs across platforms easily.

If there is anything that doesnt contain the platform name (i.e. is relying on the job name only) let me know.

Reply:
Ok, `installer_vs2019` it is. We can table the broader naming convention until we've solidified on a strategy. Agree that we should eventually have a top level job that can branch to each cross-platform deployment.

Reply:
Discussed this in another thread, but eventually we'll also want to move the signing jobs into their own pipeline to separate signing permissions and resources.

Reply:
This has already occurred; I've made the job portable enough that it can be deployed in a development environment for testing, and a separate pipeline for public simultaneously.